src/core/Akka.TestKit/Akka.TestKit.csproj

```diff
@@ -29,7 +29,7 @@
         <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftLibVersion)" />
         <PackageReference Include="Nito.AsyncEx.Coordination" Version="5.1.2" />
         <PackageReference Include="Nito.AsyncEx.Context" Version="5.1.2" />
-        <PackageReference Include="System.Linq.Async" Version="$(MicrosoftLibVersion)" />
+        <PackageReference Include="System.Linq.Async" Version="6.0.1" />
```

Comment:
System.Linq.Async is actually part of Reactive Extensions (Rx) so it doesn't ship at the normal / same cadences as the rest of the MSFT libraries, so we shouldn't include it in the version range.